### PR TITLE
Improve es6 cheatsheet

### PR DESCRIPTION
**Pull request by vikashvverma**
Added a few es6 features

screen shot 2016-02-04 at 12 40 59 am

IA Page : https://duck.co/ia/view/es6_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@vikashvverma Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=es6 cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/es6_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/es6.json [hunk 1]
```diff
@@ -1,132 +1,188 @@
...
-}
+{  
+   "id":"es6_cheat_sheet",
+   "name":"ES6",
+   "description":"ECMAScript6 (Javascript 2015 specification) syntax and information",
+   "metadata":{  
+      "sourceName":"Mozilla",
+      "sourceUrl":"https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference"
+   },
+   "aliases":[
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/es6.json]: Have a space after colons, fix this throughout the file (see the unmodified version).

File: share/goodie/cheat_sheets/json/es6.json [hunk 2]
```diff
@@ -1,132 +1,188 @@
...
+{  
+   "id":"es6_cheat_sheet",
+   "name":"ES6",
+   "description":"ECMAScript6 (Javascript 2015 specification) syntax and information",
+   "metadata":{  
+      "sourceName":"Mozilla",
+      "sourceUrl":"https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference"
+   },
+   "aliases":[  
+      "new javascript syntax",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/es6.json]: Get rid of this alias.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=es6 cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/es6_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/es6.json [hunk 3]
```diff
@@ -1,132 +1,187 @@
...
-        "Arrays",
-        "Proxies and Promises",
-        "Template Strings",
-        "Modules",
-        "Functions and Classes",
-        "Left-hand-side Expressions"
-    ],
-    "sections": {
-        "Declarations": [{
+{
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/es6.json]: You need to remove all your trailing whitespace.

**Comment by GuiltyDolphin** [comment]
Whoop, thanks :+1:

**Pull request merged**